docs/GETTING_STARTED.md

```diff
@@ -58,6 +58,29 @@ uv run pytest -c custom-pytest.ini
 uv run pytest -k test_name
 ```
 
+### Running smoke/sanity/tier1/tier2 test suites
+
+The test suite is organized into different categories based on their purpose and complexity:
```

Comment:
I don't understand this one. Are you taking about the test repo structure or markers?